### ISSUE TITLE
EMD missing?

### ISSUE THREAD

**Issue by leinxx** [fanhqme/PointSetGeneration#3]
It seems that you are using nearest neighbour distance to calculate loss instead of earth mover's distance in your code, which is inconsistent with your paper section 4.3. Am I missing something here? 

Thanks,

Lei

**Comment by ThibaultGROUEIX** [comment]
I haven't looked into details but it's probably an implementation using the Chamfer Distance, which relies on nearest neighbours. I think they mention the Chamfer Distance it in the paper.

**Comment by pclucas14** [comment]
Yes, they mention both EMD and Chamfer. It would be nice to also have the (1 + \epsilon) EMD approximation (mentioned in the paper) available. Any plans on releasing it ? 

Thanks

**Comment by jcjohnson** [comment]
Any update on this?

**Comment by ghost** [comment]
I think you can implement it by yourself. The EMD op is available from PointNet++ related papers.
